documentation/docs/framework/index.md

```diff
@@ -6,7 +6,7 @@ slug: framework.html
 
 ![intro_gif](../images/intro_gif.gif)
 
-[![version badge](https://img.shields.io/maven-central/v/io.kotest/kotest-framework-engine.svg?label=release)](https://search.maven.org/search?q=kotest)
+[![version badge](https://img.shields.io/maven-central/v/io.kotest/kotest-framework-engine.svg?label=release)](https://search.maven.org/search?q=g:io.kotest%20OR%20g:io.kotest.extensions)
```

Comment:
Note: I have chosen a link here which covers the groups `io.kotest` and `io.kotest.extensions`.
However, since the extensions use a different versioning scheme, should this only cover `io.kotest`?